iidm/iidm-api/src/main/java/com/powsybl/iidm/network/DanglingLine.java

```diff
@@ -277,7 +281,16 @@ default Generation getGeneration() {
     String getUcteXnodeCode();
 
     default Boundary getBoundary() {
-        return new DanglingLineBoundaryImpl(this);
+        throw new PowsyblException("This method must be implemented");
+        // TODO(Luma): Comment for reviewers:
```

Comment:
we try to avoid runtime exception for this kind of API change. People using this default implementation do not manipulate directly boundaries, they just use it via CGMES import/export for example and a new wrapper everytime is enough. A warning migh be added tho.